source/brailleDisplayDrivers/hims.py

```diff
@@ -291,7 +296,7 @@ def __init__(self, port="auto"):
 					case bdDetect.DeviceType.CUSTOM:
 						# onReceiveSize based on max packet size according to USB endpoint information.
 						self._dev = hwIo.Bulk(port, 0, 1, self._onReceive, onReceiveSize=64)
-					case _:
+					case bdDetect.DeviceType.SERIAL:

```

Comment:
please add a new case for `case _:` where an error is raised or logged

Reply:
I add log.error for case_ . Thanks.